<launch>

    <arg name="config" default="$(find kitaf_navigation_ros_tool)/launch/params/path_planning_parameters.yaml"/>
    <arg name="nodelet_manager_name" default="nodelet_manager_path_planning"/>
    <arg name="start_nodelet_manager" default="true"/>
    <arg name="nodelet_name" default="path_planning"/>

    <!-- Nodelet manager (if enabled) -->
    <node pkg="nodelet" type="nodelet" name="$(arg nodelet_manager_name)" args="manager" output="screen"
          if="$(arg start_nodelet_manager)"/>

    <!-- PathPlanning -->
    <node pkg="nodelet" type="nodelet" name="$(arg nodelet_name)"
          args="load kitaf_navigation_ros_tool/PathPlanningNodelet $(arg nodelet_manager_name)" output="screen" required="true">
        <rosparam command="load" file="$(arg config)"/>
    </node>

</launch>
